src/lib/slangroom/services.ts

```diff
@@ -22,48 +24,48 @@ export interface Service {
 	updated: string;
 }
 
-export const servicesContract = `
+export const getServices = async (): Promise<Response> => {
+	try {
+		const res = await slangroom.execute(
+			`
 Rule unknown ignore
 
 Given I have a 'string' named 'pb'
 When I write string 'api/collections/services/records?expand=issuer' in 'path'
 When I append 'path' to 'pb'
 Then print data
 Then I connect to 'pb' and do get and output into 'http_result'
-`;
-
-export const getServices = async (): Promise<Response> => {
-	try {
-		const res = await slangroom.execute(servicesContract, {
-			data: { pb: PB }
-		});
+`,
+			{
+				data: { pb }
+			}
```

Comment:
Is any reason to do the string concatenation in zenroom?

To me it's more readable something like

```gherkin
Rule unknown ignore

Given I have a string named 'pb'
Then print data
Then I connect to 'pb' and do get and output into 'services'
```

and pass as `data` the following:
```js
{
  "pb": PUBLIC_BACKEND_URL + 'api/collections/services/records?expand=issuer'
}
```